openmetadata-ui/src/main/resources/ui/src/rest/miscAPI.ts

```diff
@@ -264,7 +265,9 @@ export const getAggregateFieldOptions = (
   value: string,
   q: string
 ) => {
-  const withWildCardValue = value ? `.*${value}.*` : '.*';
+  const withWildCardValue = value
```

Comment:
I implemented this as the API failed if we searched for a name with some special character like `&`. I changed it in this function itself to fix this issue wherever we are using this function to fetch options.
I found this out while writing test for automation. Then checked the advanced search and quick filters for the same. This will fix that issue in all places.